src/content/tools/devtools/extensions.md

```diff
@@ -30,20 +34,29 @@ file in the root of the user's project
 This file stores per-project
 (or optionally, per user) settings for DevTools.
 
-If this file is **checked into source control**, the specified options are
-configured for the project. This means that anyone who pulls a project's
+If this file is **checked into source control**,
+the specified options are configured for the project.
+This means that anyone who pulls a project's
 source code and works on the project uses the same settings.
 
-If this file is **omitted from source control**, for example by adding
-`devtools_options.yaml` as an entry in the `.gitignore` file, then the specified
-options are configured separately for each user. Since each user or
-contributor to the project uses a local copy of the `devtools_options.yaml`
-file in this case, the specified options might differ between project contributors.
+If this file is **omitted from source control**,
+for example by adding `devtools_options.yaml`
+as an entry in the `.gitignore` file, then the specified
+options are configured separately for each user.
```

Comment:
**todo:** 
- Write in active voice per [GSG](https://developers.google.com/style/voice). 
- Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
If you **omit this file from source control**,
DevTools configures the specified options per user.
For example, if you add `devtools_options.yaml`
as an entry in the `.gitignore` file,
`git` omits the file from source control.
```